src/main/java/com/contentful/java/cma/ServicePreviewEnvironments.java

```diff
@@ -10,13 +10,12 @@
 import java.util.Map;
 
 /**
- * Tags service.
+ * Preview Environments service.
  */
-public interface ServiceTags {
-  @GET("/spaces/{spaceId}/environments/{environmentId}/tags")
```

Comment:
I didn't get what happened with Tags. If you removed class in this PR then it should appear in another place (because we need tags support)

Reply:
It's strange github behaviour. I merged last sdk master from upstream. There was new ServiceTags class. Than I created ServicePreviewEnvironments class. But github decided that I renamed my old ServiceTags class